docs/pages/reference/backends.mdx

```diff
@@ -649,7 +649,7 @@ teleport:
     # Project ID https://support.google.com/googleapi/answer/7014113?hl=en
     project_id: Example_GCP_Project_Name
 
-    # Name of the Firestore table. If it does not exist, Teleport won't start
+    # Name of the Firestore table.
```

Comment:
If it does not exist, will Teleport attempt to create one?

Reply:
Yes, this is covered in the first comment below the example configuration in the guide so I didn't add it to the comments as well

`Replace Example_GCP_Project_Name and Example_TELEPORT_FIRESTORE_TABLE_NAME with your own settings. Teleport will create the table automatically.`